MAINTAINERS.md

```diff
@@ -77,8 +87,73 @@ Make frequent project releases to the community.
 
 ### Promote Other Maintainers
 
-Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers.
+Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers. See [Becoming a Maintainer](#becoming-a-maintainer) for more information.
 
 ### Describe the Repo
 
 Make sure the repo has a well-written, accurate, and complete description. See [opensearch-project/.github#38](https://github.com/opensearch-project/.github/issues/38) for some helpful tips to describe your repo.
+
+## Becoming a Maintainer
+
+You can become a maintainer by actively [contributing](CONTRIBUTING.md) to any project, and being nominated by an existing maintainer.
+
+### Nomination
+
+Any current maintainer starts an e-mail thread (until we have a better mechanism, e-mails can usually be found via MAINTAINERS.md + DCO) with all other maintainers on that repository to discuss nomination using the template below. In order to be approved, at least 3 positive maintainer votes are necessary, and no vetoes. Any disagreements can be escalated to the repo [Admin](ADMINS.md).
```

Comment:
So we need at least 3 maintainers per repo. I think if there are fewer then there should be maintainers from other repos taking part in the vote.

Reply:
Hmmm....It may be that some repos aren't keeping their maintainer docs up to date.  :(  Which repos were you thinking of?

Reply:
It is dynamic. As stated some maintainers can take a hiatus, or be suspended for some reason. So it is a possibility unless we say something like - "All repos require a minimum of 3 maintainers" ?

Reply:
While we do try to ensure a minimum of maintainers, I think we might end up with repos with < 3. I thought it was pretty obvious that you can't have 3 people vote when there are fewer than 3 people on the list and didn't want to write that, but open for a specific edit if you think that's worth it?

Reply:
@dblock may be phrase it like that: "In order to be approved, at least 3 positive maintainer votes are necessary, and no vetoes. In rare cases when there are less than 3 maintainers, the positive votes from all maintainers are required.", wdyt?

Reply:
Added. I like it.